scanpipe/tests/data/basic-rootfs_root_filesystems.json

```diff
@@ -60,7 +59,6 @@
       "subpath": "",
       "tag": "",
       "primary_language": "",
-      "description": "shared libraries for terminal handling\n The ncurses library routines are a terminal-independent method of\n updating character screens with reasonable optimization.\n .\n This package contains the shared libraries necessary to run programs\n compiled with ncurses.",
```

Comment:
The removal of description seems to be a head-scratcher, as there are no related changes in https://github.com/nexB/scancode-toolkit/compare/v32.0.6...v32.0.7